common/test/views/support/cleaner/AmpAdCleanerTest.scala

```diff
@@ -48,6 +48,15 @@ class AmpAdCleanerTest extends FlatSpec with Matchers {
 
   }
 
+  "AmpAdCleaner" should "only add 8 ads in total" in {
```

Comment:
no need for both tests, the intention was to test that it didn't add the (n+1)th  advert, so just update it to make sure there's space for 9, but it should only add 8.